control-plane/plugin/src/resources/mod.rs

```diff
@@ -25,6 +28,8 @@ pub enum GetResources {
     Nodes,
     /// Get node with the given ID.
     Node { id: NodeId },
+    /// Get BlockDevices present on the Node. Lists usable devices by default.
```

Comment:
I think it's worth here mentioning here that the AVAILABLE field is not correct if the pool is not currently part of the control plane?

Reply:
I have added that in the args part. As an description to the --all flag. Do we need it here as well?

Reply:
I'd say move it from all to here - no sure though, up to you.

Reply:
Okay.